service/chain/protocol_state_test.go

```diff
@@ -0,0 +1,142 @@
+package chain_test
+
+import (
+	"math"
+	"testing"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage/badger/operation"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/awfm9/flow-dps/service/chain"
+)
+
+const (
+	testHeight               = 42
+	testBlockID              = "56faa737ff2b9242d1b0b7015b868aa376a623e76a65b64e8e28c7edd407113f"
+	testSeal                 = "7cef58c6e60f4c2d23cd6b7991a68297e1b6bcd95d99ad05feb0492db53dfd3c"
```

Comment:
I think at least for those two, you could probably use a `flow.Identifier{}` literal directly? What do you think?